src/vulkan/storage_buffer_descriptor.cc

```diff
@@ -119,6 +126,12 @@ Result StorageBufferDescriptor::UpdateDescriptorSet(
       descriptor_set, VK_DESCRIPTOR_TYPE_STORAGE_BUFFER, buffer_info);
 }
 
+void StorageBufferDescriptor::GetResourceInfo(ResourceInfo* info) {
```

Comment:
Can this return a ResourceInfo instead of taking a pointer?

Reply:
done